Package.swift

```diff
@@ -24,7 +24,7 @@ let package = Package(
     .package(url: "https://github.com/pointfreeco/swift-prelude.git", .revision("9161df1")),
     .package(url: "https://github.com/pointfreeco/swift-snapshot-testing.git", .revision("b4081b7")),
     .package(url: "https://github.com/bkase/DoctorPretty.git", .exact("0.3.0")),
-    .package(url: "https://github.com/IBM-Swift/BlueCryptor.git", .exact("0.8.17"))
+    .package(url: "https://github.com/IBM-Swift/BlueCryptor.git", .exact("0.8.21"))
```

Comment:
unrelated to this PR, but seems the version of `CommonCrypto` that `BlueCryptor` depended on no longer exists, and so updating `BlueCryptor` to fix that